tests/mypy/modules/success.py

```diff
@@ -230,7 +230,8 @@ class PydanticTypes(BaseModel):
     my_dir_path: DirectoryPath = Path('.')
     my_dir_path_str: DirectoryPath = '.'  # type: ignore
     # Json
-    my_json: Json = '{"hello": "world"}'
+    my_json: Json[Dict[str, str]] = '{"hello": "world"}'  # type: ignore
+    my_json_list: Json[List[str]] = '["hello", "world"]'  # type: ignore
```

Comment:
Humm, problem is this makes the output type compatibility better, but makes the input type compatibility worse.

 Hence the "type ignore" here.

Reply:
Yes, indeed. I thought that it's alright since it's kind of consistent throughout pydantic: you can even see same `type: ignore`'s above for other types.

Reply:
Yes, agreed, also presumably it allows `bytes`.

Hence why I said this is better than the status quo on balance.